### PR TITLE
Resolves #7285

### LINKED ISSUE #7285: PickList: Add localization support
**Description**
With PrimeFaces 10, localization message files are now part of the distribution. Not all components do support that. 
For example, PickList shows fixed tooltip strings like 'Add',, 'Add all' etc. for the control buttons.

As a hack for localization, the following attributes are currently available in PickList: addLabel, addAllLabel, removeLabel, removeAllLabel, moveUpLabel, moveTopLabel, moveDownLabel, moveBottomLabel.

**Describe the solution you would like**
Message resource entries should be added for these labels and used instead of the hardcoded strings.

Additional suggestion: The existing label attributes should be deprecated in version 11 and removed in version 12.

Please comment.

I will follow up with a pull request resolving this.

### PR DESCRIPTION
**Pull request by stolp**

### CODE REVIEW

File: src/main/resources/org/primefaces/Messages_de.properties [hunk 1]
```diff
@@ -30,6 +31,14 @@ primefaces.panel.aria.OPTIONS_MENU = Optionsmenü umschalten
 primefaces.panel.aria.TOGGLE = Bedienfeld ein- und ausblenden
 primefaces.password.INVALID_MATCH = {0}: Validierungsfehler.
 primefaces.password.INVALID_MATCH_detail = {0} sollte mit {1} übereinstimmen.
+primefaces.picklist.ADD = Hinzufügen
+primefaces.picklist.ADD_ALL = Alle hinzufügen
+primefaces.picklist.REMOVE = Entfernen
+primefaces.picklist.REMOVE_ALL = Alle entfernen
+primefaces.picklist.MOVE_UP = Nach oben verschieben
+primefaces.picklist.MOVE_TOP = Nach oben verschieben
```
Comment (jepsar) [inline src/main/resources/org/primefaces/Messages_de.properties]: Why is the text the same for both?

Comment (stolp) [reply src/main/resources/org/primefaces/Messages_de.properties]: My bad. It was way too late yesterday. Thanks for spotting!

Comment (stolp) [reply src/main/resources/org/primefaces/Messages_de.properties]: I resolved the translation and added your NL translation as well.

File: src/main/resources/org/primefaces/Messages_de.properties [hunk 2]
```diff
@@ -36,9 +36,9 @@ primefaces.picklist.ADD_ALL = Alle hinzufügen
 primefaces.picklist.REMOVE = Entfernen
 primefaces.picklist.REMOVE_ALL = Alle entfernen
 primefaces.picklist.MOVE_UP = Nach oben verschieben
-primefaces.picklist.MOVE_TOP = Nach oben verschieben
+primefaces.picklist.MOVE_TOP = An den Anfang verschieben
```
Comment (jepsar) [inline src/main/resources/org/primefaces/Messages_de.properties:39]: FYI in Dutch I left out the equivalent of verschieben (as with German, translations can get long compared to English).

Comment (stolp) [reply src/main/resources/org/primefaces/Messages_de.properties:39]: Your call - But as this is a tooltip there is enough room to be verbose I think

**Review by melloware — APPROVED** [review]

**Comment by jepsar** [comment]
Maybe update https://github.com/primefaces/primefaces/blob/master/docs/11_0_0/core/localization.md

**Comment by melloware** [comment]
@stolp While your updatig the 11.0 MD can you add two I forgot to add.
```
primefaces.autocomplete.aria.DROPDOWN_LABEL=Show Options
primefaces.switch.LABEL_ON = On
primefaces.switch.LABEL_OFF = Off
```

**Comment by stolp** [comment]
I'm on it.

**Comment by stolp** [comment]
Done and done.

**Review by melloware — APPROVED** [review]

**Pull request merged**